ibis-server/Dockerfile

```diff
@@ -27,10 +27,33 @@ RUN poetry install --without dev
 FROM python:3.11-slim-buster as runtime
 
 # libpq-dev is required for psycopg2
+# unixodbc-dev and freetds-dev are required for pyodbc
 RUN apt-get update  \
     && apt-get -y install libpq-dev \
+    && apt-get -y install unixodbc-dev freetds-dev tdsodbc \
```

Comment:
How about having a comment for `tdsodbc`? Maybe say it's required for MSSQL?

Reply:
Yes, freetds need tdsodbc. I will add tdsodbc to the comment.